src/components/dictionaryConcepts/proptypes.js

```diff
@@ -14,7 +14,7 @@ export const conceptsProps = {
   display_name: PropTypes.string,
   display_locale: PropTypes.string,
   version: PropTypes.string,
-  mappings: PropTypes.string,
+  mappings: PropTypes.arrayOf,
```

Comment:
Can you supply the parameter in the `arrayOf`?

Reply:
I think it is not appropriate since we are defining global proptypes here.

Reply:
@dessHub What do you mean by defining global proptypes?

Reply:
@muhozi and @Duncanian on a closer look into this, I have understood what exactly is going on here. This file was meant to avoid deep object nesting  `concepts: PropTypes.arrayOf(PropTypes.shape(conceptsProps)).isRequired,`. So to answer @Duncanian question, yes a parameter can be supplied. Taking care of it right away.